apps/server/src/modules/classes/service/mapper/class.mapper.ts

```diff
@@ -0,0 +1,50 @@
+import { ObjectId } from '@mikro-orm/mongodb';
+import { Class } from '../../domain';
+import { ClassSourceOptions } from '../../domain/class-source-options.do';
+import { ClassEntity } from '../../entity';
+
+export class ClassMapper {
```

Comment:
this should be moved into the repo folder, as its used (only) there.

Reply:
fixed